components/checkbox/checkbox-group.component.ts

```diff
@@ -79,11 +79,13 @@ export class NzCheckboxGroupComponent implements ControlValueAccessor, OnInit, O
   }
 
   ngOnInit(): void {
-    this.focusMonitor.monitor(this.elementRef, true).subscribe(focusOrigin => {
-      if (!focusOrigin) {
-        Promise.resolve().then(() => this.onTouched());
-      }
-    });
+    this.focusMonitor.monitor(this.elementRef, true)
+      .pipe(takeUntil(this.destroy$))
```

Comment:
Thanks for your contribution, I see that just unsubscribing doesn't make it stop monitoring, we need to call `stopMonitoring` explicitly. In this case here we can safely do it, could you modify them?

Reply:
Hi @hsuanxyz, as far as I can see there is already a `stopMonitoring` call inside `ngOnDestroy` of this component. 
Some of the components were calling `stopMonitoring` but didn't `unsubscribe` from the stream.

Reply:
Oh, I got. @raizemm  would you plz rebase this pr onto master.

Reply:
Done. @hsuanxyz